packages/email-pcd/package.json

```diff
@@ -3,24 +3,32 @@
   "version": "0.3.0",
   "license": "GPL-3.0-or-later",
   "main": "./dist/index.js",
-  "types": "./src/index.ts",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
   "files": [
-    "./src/*",
-    "./dist/*",
+    "dist",
     "./README.md",
     "./LICENSE"
   ],
   "scripts": {
     "lint": "eslint \"**/*.ts{,x}\"",
-    "build": "tsup src/index.ts",
+    "build": "tsup src/index.ts --format cjs,esm --dts --clean",
     "dev": "tsup src/index.ts --watch",
     "typecheck": "yarn tsc --noEmit",
```

Comment:
question: here and elsewhere, if we're no longer using the `typecheck` command, should we remove it?